doc/ch-image.rst

```diff
@@ -590,6 +590,18 @@ Options:
 
     If no colon present in the name, append :code:`:latest`.
 
+  :code:`--target STAGE`
+    Set the upper bound stage of a multistage build. Instructions up to, and
+    including, the target build stage are executed. All subsequent instructions
+    are ignored. :code:`STAGE` can be specified as the stage:
+
+    1. number, e.g., :code:`0`, :code:`1`...:code:`n`;
+
+    2. alias, e.g., :code:`alp` as declared by
+    :code:`FROM alpine:3.16 as alp`; or
+
+    3. base name, e.g., :code:`alpine:3.16`
```

Comment:
1. This is much more clear than Docker's docs for the option. Nice work.
2. It doesn't render correctly, unfortunately.